cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -1184,10 +1478,10 @@ makeShelleyTransactionBody era@ShelleyBasedEraAllegra
              TxFeeImplicit era'  -> case era' of {}
              TxFeeExplicit _ fee -> toShelleyLovelace fee)
           (Allegra.ValidityInterval {
-             Allegra.invalidBefore    = case lowerBound of
```

Comment:
This is the Allegra era, so we should leave `invalidBefore` and `invalidHereafter` qualified